.pre-commit-config.yaml

```diff
@@ -0,0 +1,18 @@
+ci:
+  autofix_prs: false
+  autoupdate_schedule: 'monthly'
+
+repos:
+  - repo: https://github.com/pre-commit/pre-commit-hooks
+    rev: v6.0.0
+    hooks:
+      - id: check-added-large-files
+        args: ["--enforce-all", "--maxkb=100"]
```

Comment:
[The default file size limit is 500 kB](https://github.com/pre-commit/pre-commit-hooks?tab=readme-ov-file#check-added-large-files), but here 100 kB seems to be plenty.